client/dive-common/components/Sidebar.vue

```diff
@@ -170,6 +188,7 @@ export default defineComponent({
           @back="swapTabs"
           @commit-merge="commitMerge"
           @create-group="groupAdd"
+          @delete-selected-tracks="deleteSelectedTracks"
```

Comment:
I think the default behavior for multi deletion of selected tracks is to deselect without a previous/next selection.  It may be nice to wrap this in the toggle for the `swapTabs`.  That way with the user deletes all of their selected tracks it immediately goes back to the TypeList/TrackList view.

Reply:
This should be fixed by ce710d9a